modules/logging-in-by-using-the-web-console.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="logging-in-by-using-the-web-console_{context}"]
+= Logging in to the cluster by using the web console
+
+You can log into your cluster as `kubeadmin` by using the {product-title} web console after the Console Operator becomes available.
```

Comment:
What's `kubeadmin`?

> You can log into your cluster as the `kubeadmin` user...

Reply:
Thank you for highlighting this. I have subsequently updated the introduction to this module as follows:

"The `kubeadmin` user exists by default after an {product-title} installation. You can log into your cluster as the `kubeadmin` user by using the {product-title} web console after the Console Operator becomes available."